Failing Unit Tests - develop Branch

<!-- For replicatable bugs or feature requests GitHub is the right place. The [Foundation Forum](http://foundation.zurb.com/forum) would be a great place to seek some help with general questions or help with your code. There you can reach out to the community to share your insights or ask questions. -->

#### How to reproduce this bug:

1. Execute `npm test`

#### What should happen:

* Unit tests should pass

#### What happened instead:

~~~
> foundation-sites@6.4.3 test C:\git\foundation-sites
> npm run test:sass && npm run test:javascript:phantomjs


> foundation-sites@6.4.3 test:sass C:\git\foundation-sites
> mocha test/sass/test_sass.js

npm : WARNING: breakpoint(): "xxxxlarge" is not defined in your $breakpoints 
setting.
At line:1 char:1
+ npm test 1> output 2>&1
+ ~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (WARNING: breakp...points setting. 
   :String) [], RemoteException
    + FullyQualifiedErrorId : NativeCommandError
 
Backtrace:
	scss/util/_breakpoint.scss:73, in function `breakpoint`
	test/sass/_breakpoint.scss:97, in mixin `@content`
	node_modules/sass-true/sass/true/_tests.scss:17, in mixin `test`
	test/sass/_breakpoint.scss:96, in mixin `@content`
	node_modules/sass-true/sass/true/_modules.scss:17, in mixin `test-module`
	test/sass/_breakpoint.scss:6





  Breakpoint
    ΓêÜ Breakpoint (Named to Em) [function]
    ΓêÜ Breakpoint (Rem/Px to Em) [function]
    ΓêÜ Breakpoint (Only Range) [function]
    ΓêÜ Breakpoint (Named Down Range) [function]
    ΓêÜ Breakpoint (Value Down Range) [function]
    ΓêÜ Breakpoint (Empty String) [function]
    ΓêÜ Breakpoint (Orientation/Retina) [function]
    ΓêÜ Breakpoint (Unknown Value) [function]
    ΓêÜ Map Serialize [function]
    ΓêÜ Map Next [function]
    ΓêÜ Get Breakpoint Value [function]

  Color
    ΓêÜ Foreground (Black) [function]
    ΓêÜ Foreground (White) [function]
    ΓêÜ Smart Scale (Darken) [function]
    ΓêÜ Smart Scale (Lighten) [function]

  Selector
    ΓêÜ Selector [function]
    ΓêÜ Selector (with modifiers) [function]

  Units
    ΓêÜ Strip Units [function]
    ΓêÜ Convert To Rem [function]
    ΓêÜ Rem Calculator [function]
    ΓêÜ Breakpoint To Em [function]

  Value
    ΓêÜ Value (Not Falsey) [function]
    ΓêÜ Value (Falsey) [function]
    ΓêÜ Get Side [function]
    ΓêÜ Get Border Value [function]
    ΓêÜ Map Deep Get [function]
    ΓêÜ Map Safe Get [function]

  Components
    ΓêÜ Ratio to Percentage [function]
    ΓêÜ Grid Column [function]
    ΓêÜ Flex Grid Column [function]


  30 passing (19ms)


> foundation-sites@6.4.3 test:javascript:phantomjs C:\git\foundation-sites
> npm run test:javascript:transpile && mocha-phantomjs --ignore-resource-errors test/javascript/index.html


> foundation-sites@6.4.3 test:javascript:transpile C:\git\foundation-sites
> gulp sass:foundation && gulp test:transpile-js

[17:08:07] Using gulpfile C:\git\foundation-sites\gulpfile.js
[17:08:07] Starting 'sass:deps'...
[17:08:07] Finished 'sass:deps' after 108 ms
[17:08:07] Starting 'sass:foundation'...
WARNING: 
null was passed to rem-calc(), which is not a number.

Backtrace:
	scss/util/_unit.scss:78, in function `-zf-to-rem`
	scss/util/_unit.scss:47, in function `rem-calc`
	scss/xy-grid/_cell.scss:84, in function `if`
	scss/xy-grid/_cell.scss:84, in mixin `-xy-cell-properties`
	scss/xy-grid/_cell.scss:167, in mixin `xy-cell-static`
	scss/xy-grid/_classes.scss:275, in mixin `xy-vertical-grid-classes`
	scss/xy-grid/_classes.scss:470, in mixin `foundation-xy-grid-classes`
	scss/foundation.scss:84, in mixin `foundation-everything`
	assets/foundation-prototype.scss:3



WARNING: 
null was passed to rem-calc(), which is not a number.

Backtrace:
	scss/util/_unit.scss:78, in function `-zf-to-rem`
	scss/util/_unit.scss:47, in function `rem-calc`
	scss/xy-grid/_cell.scss:84, in function `if`
	scss/xy-grid/_cell.scss:84, in mixin `-xy-cell-properties`
	scss/xy-grid/_cell.scss:167, in mixin `xy-cell-static`
	scss/xy-grid/_classes.scss:275, in mixin `xy-vertical-grid-classes`
	scss/xy-grid/_classes.scss:470, in mixin `foundation-xy-grid-classes`
	scss/foundation.scss:84, in mixin `foundation-everything`
	assets/foundation-rtl.scss:5



WARNING: 
null was passed to rem-calc(), which is not a number.

Backtrace:
	scss/util/_unit.scss:78, in function `-zf-to-rem`
	scss/util/_unit.scss:47, in function `rem-calc`
	scss/xy-grid/_cell.scss:84, in function `if`
	scss/xy-grid/_cell.scss:84, in mixin `-xy-cell-properties`
	scss/xy-grid/_cell.scss:167, in mixin `xy-cell-static`
	scss/xy-grid/_classes.scss:275, in mixin `xy-vertical-grid-classes`
	scss/xy-grid/_classes.scss:470, in mixin `foundation-xy-grid-classes`
	scss/foundation.scss:84, in mixin `foundation-everything`
	assets/foundation.scss:3



[17:08:10] Finished 'sass:foundation' after 3.59 s

scss\_global.scss
  169:7  warning  Vendor prefixes should not be used  no-vendor-prefixes
  170:7  warning  Vendor prefixes should not be used  no-vendor-prefixes

Γ£û 2 problems (0 errors, 2 warnings)


scss\components\_accordion-menu.scss
   54:3  warning  Pseudo-class should be nested within its parent Class                 force-pseudo-nesting
   65:1  warning  Space expected between blocks                                         empty-line-between-blocks
   65:3  warning  Pseudo-element should be nested within its parent Type-selector       force-pseudo-nesting
   65:3  warning  Class should be nested within its parent Class                        force-element-nesting
   66:5  warning  Expected `right`, found `left`                                        property-sort-order
   67:5  warning  Expected `left`, found `right`                                        property-sort-order
   69:1  warning  Space expected between blocks                                         empty-line-between-blocks
   69:3  warning  Class should be nested within its parent Class                        force-element-nesting
   69:3  warning  Pseudo-element should be nested within its parent Type-selector       force-pseudo-nesting
   97:1  warning  Space expected between blocks                                         empty-line-between-blocks
   97:5  warning  Type-selector should be nested within its parent Class                force-element-nesting
  101:5  warning  Class should be nested within its parent Class                        force-element-nesting
  106:7  warning  Class should be nested within its parent Class                        force-element-nesting
  114:7  warning  Attribute-selector should be nested within its parent Class           force-attribute-nesting
  114:7  warning  Pseudo-element should be nested within its parent Type-selector       force-pseudo-nesting
  121:3  warning  Type-selector should be nested within its parent Class                force-element-nesting
  141:5  warning  Expected `width`, found `cursor`                                      property-sort-order
  143:5  warning  Expected `height`, found `width`                                      property-sort-order
  144:5  warning  Expected `cursor`, found `height`                                     property-sort-order
  163:3  warning  Pseudo-element should be nested within its parent Attribute-selector  force-pseudo-nesting
  163:3  warning  Attribute-selector should be nested within its parent Class           force-attribute-nesting

Γ£û 21 problems (0 errors, 21 warnings)


scss\components\_accordion.scss
  56:1  warning  Space expected between blocks                                 empty-line-between-blocks
  93:3  warning  Pseudo-class should be nested within its parent Pseudo-class  force-pseudo-nesting

Γ£û 2 problems (0 errors, 2 warnings)


scss\components\_breadcrumbs.scss
  85:7  warning  Pseudo-element should be nested within its parent Pseudo-class  force-pseudo-nesting

Γ£û 1 problem (0 errors, 1 warning)


scss\components\_button-group.scss
  116:7   warning  Pseudo-class should be nested within its parent Pseudo-class  force-pseudo-nesting
  122:9   warning  Pseudo-class should be nested within its parent Pseudo-class  force-pseudo-nesting
  123:14  warning  Pseudo-class should be nested within its parent Pseudo-class  force-pseudo-nesting
  163:19  warning  Whitespace required before {                                  space-before-brace
  215:7   warning  Type-selector should be nested within its parent Class        force-element-nesting
  241:11  warning  Type-selector should be nested within its parent Class        force-element-nesting

Γ£û 6 problems (0 errors, 6 warnings)


scss\components\_button.scss
   98:3   warning  Expected `border`, found `font-family`                              property-sort-order
   99:3   warning  Expected `border-radius`, found `font-weight`                       property-sort-order
  112:3   warning  Vendor prefixes should not be used                                  no-vendor-prefixes
  113:3   warning  Expected `transition`, found `border`                               property-sort-order
  114:3   warning  Expected `font-family`, found `border-radius`                       property-sort-order
  115:3   warning  Expected `font-size`, found `transition`                            property-sort-order
  116:3   warning  Expected `font-weight`, found `font-size`                           property-sort-order
  144:22  warning  No empty blocks allowed                                             no-empty-rulesets
  199:3   warning  Pseudo-class should be nested within its parent Class               force-pseudo-nesting
  199:21  warning  Pseudo-class should be nested within its parent Attribute-selector  force-pseudo-nesting
  200:3   warning  Pseudo-class should be nested within its parent Class               force-pseudo-nesting
  200:21  warning  Pseudo-class should be nested within its parent Attribute-selector  force-pseudo-nesting
  235:3   warning  Pseudo-class should be nested within its parent Class               force-pseudo-nesting
  235:21  warning  Pseudo-class should be nested within its parent Attribute-selector  force-pseudo-nesting
  236:3   warning  Pseudo-class should be nested within its parent Class               force-pseudo-nesting
  236:21  warning  Pseudo-class should be nested within its parent Attribute-selector  force-pseudo-nesting
  270:3   warning  Pseudo-class should be nested within its parent Class               force-pseudo-nesting
  270:21  warning  Pseudo-class should be nested within its parent Attribute-selector  force-pseudo-nesting
  271:3   warning  Pseudo-class should be nested within its parent Class               force-pseudo-nesting
  271:21  warning  Pseudo-class should be nested within its parent Attribute-selector  force-pseudo-nesting
  414:5   warning  Pseudo-element should be nested within its parent Class             force-pseudo-nesting
  420:1   warning  Space expected between blocks                                       empty-line-between-blocks
  420:3   warning  Class should be nested within its parent Type-selector              force-element-nesting
  420:4   warning  Qualifying elements are not allowed for class selectors             no-qualifying-elements

Γ£û 24 problems (0 errors, 24 warnings)


scss\components\_card.scss
  84:5  warning  Expected `display`, found `flex`  property-sort-order
  85:5  warning  Expected `flex`, found `display`  property-sort-order

Γ£û 2 problems (0 errors, 2 warnings)


scss\components\_drilldown.scss
   48:1  warning  Space expected between blocks                                    empty-line-between-blocks
   53:7  warning  Mixins should come before declarations                           mixins-before-declarations
   57:3  warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting
   57:3  warning  Class should be nested within its parent Class                   force-element-nesting
   58:5  warning  Expected `right`, found `left`                                   property-sort-order
   59:5  warning  Expected `left`, found `right`                                   property-sort-order
   60:5  warning  Mixins should come before declarations                           mixins-before-declarations
   63:3  warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting
   63:3  warning  Class should be nested within its parent Class                   force-element-nesting
   66:5  warning  Mixins should come before declarations                           mixins-before-declarations
  120:5  warning  Class should be nested within its parent Class                   force-element-nesting
  131:7  warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting
  138:9  warning  Duplicate properties are not allowed within a block              no-duplicate-properties

Γ£û 13 problems (0 errors, 13 warnings)


scss\components\_dropdown-menu.scss
   67:3   warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting
   71:3   warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting
   74:5   warning  Mixins should come before declarations                           mixins-before-declarations
   77:3   warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting
   84:5   warning  Class should be nested within its parent Type-selector           force-element-nesting
   84:9   warning  Qualifying elements are not allowed for class selectors          no-qualifying-elements
   92:5   warning  Class should be nested within its parent Type-selector           force-element-nesting
   92:9   warning  Qualifying elements are not allowed for class selectors          no-qualifying-elements
  101:7   warning  Class should be nested within its parent Type-selector           force-element-nesting
  101:11  warning  Qualifying elements are not allowed for class selectors          no-qualifying-elements
  106:7   warning  Class should be nested within its parent Type-selector           force-element-nesting
  106:7   warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting
  106:11  warning  Qualifying elements are not allowed for class selectors          no-qualifying-elements
  122:11  warning  Expected `top`, found `right`                                    property-sort-order
  123:11  warning  Expected `right`, found `left`                                   property-sort-order
  124:11  warning  Expected `left`, found `top`                                     property-sort-order
  146:3   warning  Class should be nested within its parent Class                   force-element-nesting
  164:5   warning  Class should be nested within its parent Class                   force-element-nesting
  187:7   warning  Class should be nested within its parent Class                   force-element-nesting
  195:3   warning  Class should be nested within its parent Class                   force-element-nesting
  206:5   warning  Pseudo-element should be nested within its parent Type-selector  force-pseudo-nesting

Γ£û 21 problems (0 errors, 21 warnings)


scss\components\_dropdown.scss
  46:3  warning  Expected `display`, found `width`               property-sort-order
  47:3  warning  Expected `width`, found `padding`               property-sort-order
  49:3  warning  Expected `padding`, found `visibility`          property-sort-order
  50:3  warning  Expected `visibility`, found `display`          property-sort-order
  64:5  warning  Expected `display`, found `visibility`          property-sort-order
  65:5  warning  Expected `visibility`, found `display`          property-sort-order
  75:5  warning  Class should be nested within its parent Class  force-element-nesting

Γ£û 7 problems (0 errors, 7 warnings)


scss\components\_flex.scss
  83:7  warning  Class should be nested within its parent Class  force-element-nesting

Γ£û 1 problem (0 errors, 1 warning)

[17:08:19] Using gulpfile C:\git\foundation-sites\gulpfile.js
[17:08:19] Starting 'javascript:plugin-core'...
[17:08:19] Starting 'javascript:deps'...
[17:08:20] Finished 'javascript:deps' after 288 ms
[17:08:20] Version: webpack 3.5.5
             Asset     Size  Chunks             Chunk Names
foundation.core.js  30.2 kB       0  [emitted]  foundation.core
[17:08:20] Finished 'javascript:plugin-core' after 1.11 s
[17:08:20] Starting 'javascript:plugins'...
[17:08:25] Version: webpack 3.5.5
                                Asset     Size  Chunks             Chunk Names
       foundation.responsiveToggle.js  12.1 kB      15  [emitted]  foundation.responsiveToggle
                foundation.tooltip.js  42.8 kB       0  [emitted]  foundation.tooltip
               foundation.dropdown.js  43.5 kB       2  [emitted]  foundation.dropdown
                 foundation.slider.js  44.4 kB       3  [emitted]  foundation.slider
         foundation.responsiveMenu.js  13.2 kB       4  [emitted]  foundation.responsiveMenu
              foundation.offcanvas.js  38.8 kB       5  [emitted]  foundation.offcanvas
                 foundation.sticky.js    35 kB       6  [emitted]  foundation.sticky
                 foundation.reveal.js    38 kB       7  [emitted]  foundation.reveal
foundation.responsiveAccordionTabs.js    18 kB       8  [emitted]  foundation.responsiveAccordionTabs
           foundation.dropdownMenu.js    24 kB       9  [emitted]  foundation.dropdownMenu
              foundation.drilldown.js  31.5 kB      10  [emitted]  foundation.drilldown
              foundation.equalizer.js  18.3 kB      11  [emitted]  foundation.equalizer
          foundation.accordionMenu.js  19.1 kB      12  [emitted]  foundation.accordionMenu
                foundation.toggler.js  22.2 kB      13  [emitted]  foundation.toggler
                   foundation.tabs.js  22.1 kB      14  [emitted]  foundation.tabs
                  foundation.orbit.js  25.8 kB       1  [emitted]  foundation.orbit
               foundation.magellan.js  15.8 kB      16  [emitted]  foundation.magellan
            foundation.interchange.js  13.4 kB      17  [emitted]  foundation.interchange
              foundation.accordion.js  17.4 kB      18  [emitted]  foundation.accordion
           foundation.smoothScroll.js    11 kB      19  [emitted]  foundation.smoothScroll
                  foundation.abide.js  32.7 kB      20  [emitted]  foundation.abide
          foundation.util.triggers.js  14.8 kB      21  [emitted]  foundation.util.triggers
            foundation.util.motion.js  7.22 kB      22  [emitted]  foundation.util.motion
          foundation.util.keyboard.js  9.82 kB      23  [emitted]  foundation.util.keyboard
             foundation.util.timer.js  5.12 kB      24  [emitted]  foundation.util.timer
              foundation.util.nest.js   5.9 kB      25  [emitted]  foundation.util.nest
        foundation.util.mediaQuery.js  10.6 kB      26  [emitted]  foundation.util.mediaQuery
       foundation.util.imageLoader.js  5.14 kB      27  [emitted]  foundation.util.imageLoader
               foundation.util.box.js  12.8 kB      28  [emitted]  foundation.util.box
             foundation.util.touch.js  9.19 kB      29  [emitted]  foundation.util.touch
[17:08:25] Finished 'javascript:plugins' after 4.72 s
[17:08:25] Starting 'javascript:foundation'...
[17:08:29] Version: webpack 3.5.5
        Asset    Size  Chunks                    Chunk Names
foundation.js  439 kB       0  [emitted]  [big]  foundation
[17:08:29] Finished 'javascript:foundation' after 3.67 s
[17:08:29] Starting 'test:transpile-js'...
[17:08:30] Finished 'test:transpile-js' after 692 ms

[0m[0m
[0m  Abide[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element & plugin options[0m
[2K[0G    [32m  Γ£ô[0m[90m the options are recursively merged[0m
[0m    validateInput()[0m
[2K[0G    [32m  Γ£ô[0m[90m returns true for hidden inputs[0m
[2K[0G    [32m  Γ£ô[0m[90m returns true for inputs with [data-abide-ignore][0m
[2K[0G    [32m  Γ£ô[0m[90m returns true for checked checkboxes[0m
[2K[0G    [32m  Γ£ô[0m[90m returns false for unchecked checkboxes[0m
[2K[0G    [32m  Γ£ô[0m[90m returns true for selected select option[0m
[2K[0G    [32m  Γ£ô[0m[90m returns false for unselected select option[0m
[0m    addErrorClasses()[0m
[2K[0G    [32m  Γ£ô[0m[90m adds aria-invalid attribute to element[0m
[0m    addGlobalErrorA11yAttributes()[0m
[2K[0G    [32m  Γ£ô[0m[90m adds [aria-live] attribute on element[0m
[0m    addA11yAttributes()[0m
[2K[0G    [31m  1) adds [aria-describedby] attribute to field and [for] attribute to form error[0m
[2K[0G    [31m  2) adds attributes and ids when no id is set[0m
[0m    removeErrorClasses()[0m
[2K[0G    [32m  Γ£ô[0m[90m removes aria-invalid attribute from element[0m
[0m    removeRadioErrorClasses()[0m
[2K[0G    [32m  Γ£ô[0m[90m removes aria-invalid attribute from radio group[0m
[0m    resetForm()[0m
[2K[0G    [32m  Γ£ô[0m[90m removes aria-invalid attribute from elements[0m

[0m  Accordion[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    up()[0m
[2K[0G    [32m  Γ£ô[0m[90m closes the targeted container if allowAllClosed is true[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles attributes of title of the targeted container[0m
[2K[0G    [32m  Γ£ô[0m[90m not closes the open container if allowAllClosed is false[0m
[0m    down()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the targeted container[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles attributes of title of the targeted container[0m
[2K[0G    [32m  Γ£ô[0m[90m closes open container if multiExpand is false[0m[31m (458ms)[0m
[2K[0G    [32m  Γ£ô[0m[90m not closes open container if multiExpand is true[0m
[0m    toggle()[0m
[2K[0G    [32m  Γ£ô[0m[90m closes the only open container if allowAllClosed is true[0m
[2K[0G    [32m  Γ£ô[0m[90m not closes the only open container if allowAllClosed is false[0m
[0m    keyboard events[0m
[2K[0G    [32m  Γ£ô[0m[90m opens next panel on ARROW_DOWN[0m
[2K[0G    [32m  Γ£ô[0m[90m opens previous panel on ARROW_UP[0m
[2K[0G    [32m  Γ£ô[0m[90m opens related panel on ENTER[0m
[2K[0G    [32m  Γ£ô[0m[90m opens related panel on SPACE[0m

[0m  Accordion Menu[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    up()[0m
[2K[0G    [32m  Γ£ô[0m[90m closes the targeted submenu[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles attributes of title of the targeted container[0m
[2K[0G    [32m  Γ£ô[0m[90m fires up.zf.accordionMenu event[0m[31m (410ms)[0m
[0m    down()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the targeted submenu[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles attributes of title of the targeted submenu[0m
[2K[0G    [32m  Γ£ô[0m[90m closes open submenu if multiOpen is false[0m
[2K[0G    [32m  Γ£ô[0m[90m not closes open submenu if multiOpen is true[0m
[2K[0G    [32m  Γ£ô[0m[90m fires down.zf.accordionMenu event[0m[31m (210ms)[0m
[0m    toggle()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens a closed container[0m
[2K[0G    [32m  Γ£ô[0m[90m closes an open container[0m
[0m    hideAll()[0m
[2K[0G    [32m  Γ£ô[0m[90m closes all accordions[0m

[0m  Drilldown Menu[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores additional elements[0m
[0m    prepareMenu()[0m
[2K[0G    [32m  Γ£ô[0m[90m wraps the submenus[0m
[2K[0G    [32m  Γ£ô[0m[90m adds ARIA attributes[0m
[0m    show()[0m
[2K[0G    [32m  Γ£ô[0m[90m shows the given submenu[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m fires open.zf.drilldown event[0m
[0m    hide()[0m
[2K[0G    [32m  Γ£ô[0m[90m hides the given submenu[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m fires hide.zf.drilldown event[0m
[0m    hideAll()[0m
[2K[0G    [32m  Γ£ô[0m[90m hides all submenus[0m
[2K[0G    [32m  Γ£ô[0m[90m fires closed.zf.drilldown event[0m
[0m    back()[0m
[2K[0G    [32m  Γ£ô[0m[90m hides current submenu[0m
[2K[0G    [32m  Γ£ô[0m[90m shows parent submenu[0m
[0m    keyboard events[0m
    [36m  - does not trap focus on root element going down[0m
    [36m  - does not trap focus on root element going up[0m
[2K[0G    [32m  Γ£ô[0m[90m closes current sub menu using ESC[0m
[2K[0G    [32m  Γ£ô[0m[90m moves focus to next element on TAB[0m
[2K[0G    [32m  Γ£ô[0m[90m moves focus to previous element on TAB[0m
[2K[0G    [32m  Γ£ô[0m[90m moves focus to next element on ARROW_DOWN[0m
[2K[0G    [32m  Γ£ô[0m[90m moves focus to previous element on ARROW_UP[0m
[2K[0G    [32m  Γ£ô[0m[90m opens child element on ARROW_RIGHT[0m
[2K[0G    [32m  Γ£ô[0m[90m focuses parent link if parentLink is true[0m[31m (523ms)[0m
[2K[0G    [32m  Γ£ô[0m[90m closes child element on ARROW_LEFT[0m

[0m  Dropdown[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element & plugin options[0m
[0m    open()[0m
[2K[0G    [32m  Γ£ô[0m[90m traps focus if trapFocus option is true[0m
[0m    close()[0m
[2K[0G    [32m  Γ£ô[0m[90m releases focus if trapFocus option is true[0m
[0m    inferred positioning[0m
[2K[0G    [32m  Γ£ô[0m[90m default orientation should be bottom[0m
[2K[0G    [32m  Γ£ô[0m[90m gets right alignment from float-right[0m
[0m    closeOnClick option[0m
[2K[0G    [32m  Γ£ô[0m[90m not closes a dropdown by clicking on the dropdown if closeOnClick option is true[0m
[0m    keyboard events[0m
[2K[0G    [32m  Γ£ô[0m[90m opens Dropdown on SPACE[0m
[2K[0G    [32m  Γ£ô[0m[90m focuses Dropdown on SPACE[0m
[2K[0G    [32m  Γ£ô[0m[90m does not focus Dropdown when anchor is an input[0m

[0m  Dropdown Menu[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    keyboard events[0m
[2K[0G    [32m  Γ£ô[0m[90m closes current sub menu using ESCAPE[0m
[0m      horizontal[0m
[2K[0G      [32m  Γ£ô[0m[90m moves focus to next tab on ARROW_RIGHT[0m
[2K[0G      [32m  Γ£ô[0m[90m moves focus to previous tab on ARROW_LEFT[0m
[2K[0G      [32m  Γ£ô[0m[90m opens child element of tab on ARROW_DOWN[0m
[2K[0G      [32m  Γ£ô[0m[90m moves focus to previous sub element on ARROW_UP[0m
[2K[0G      [32m  Γ£ô[0m[90m opens child element of sub menu on ARROW_RIGHT[0m
[0m      vertical[0m
[2K[0G      [32m  Γ£ô[0m[90m moves focus to next tab on ARROW_DOWN[0m
[2K[0G      [32m  Γ£ô[0m[90m moves focus to previous tab on ARROW_UP[0m
[2K[0G      [32m  Γ£ô[0m[90m opens child element of tab on ARROW_RIGHT[0m
[2K[0G      [32m  Γ£ô[0m[90m moves focus to previous sub element on ARROW_UP[0m
[2K[0G      [32m  Γ£ô[0m[90m opens child element of sub menu on ARROW_RIGHT[0m

[0m  Interchange[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    replace()[0m
[2K[0G    [32m  Γ£ô[0m[90m replaces src attribute of img[0m
[2K[0G    [32m  Γ£ô[0m[90m replaces background style of divs[0m
[2K[0G    [32m  Γ£ô[0m[90m replaces contents of div with templates[0m
[2K[0G    [32m  Γ£ô[0m[90m fires replaced.zf.interchange event[0m
[0m    reflow()[0m
[2K[0G    [32m  Γ£ô[0m[90m calls replace for given media query[0m
[0m    generateRules()[0m
[2K[0G    [32m  Γ£ô[0m[90m extracts rules from the plugin element[0m
[2K[0G    [32m  Γ£ô[0m[90m extracts special queries from the plugin element[0m
[0m    addBreakpoints()[0m
[2K[0G    [32m  Γ£ô[0m[90m adds Foundation breakpoints to special queries[0m
[0m    events()[0m
[2K[0G    [32m  Γ£ô[0m[90m calls reflow on viewport size change once[0m[33m (60ms)[0m

[0m  Magellan[0m
[0m    scrollToLoc()[0m
[2K[0G    [32m  Γ£ô[0m[90m scrolls the selected element into viewport[0m[31m (223ms)[0m
[2K[0G    [32m  Γ£ô[0m[90m fails gracefully when target does not exist[0m

[0m  Off Canvas[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m finds triggers for the Off Canvas[0m
[2K[0G    [32m  Γ£ô[0m[90m sets ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m closes Off Canvas on outside click if closeOnClick option is true[0m
[0m    open()[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m adds active classes[0m
[2K[0G    [32m  Γ£ô[0m[90m focusses Off Canvas if autoFocus option is true[0m
[2K[0G    [32m  Γ£ô[0m[90m traps focus if trapFocus option is true[0m[31m (538ms)[0m
[2K[0G    [32m  Γ£ô[0m[90m fires opened.zf.offcanvas event[0m
[0m    close()[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m removes active classes[0m
[2K[0G    [32m  Γ£ô[0m[90m fires closed.zf.offcanvas event[0m
[2K[0G    [32m  Γ£ô[0m[90m releases focus if trapFocus option is true[0m
[0m    toggle()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens a closed Off Canvas[0m
[2K[0G    [32m  Γ£ô[0m[90m closes an open Off Canvas[0m
[0m    keyboard events[0m
[2K[0G    [32m  Γ£ô[0m[90m closes Off Canvas on ESCAPE[0m

[0m  Orbit[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m shows the first slide[0m
[2K[0G    [32m  Γ£ô[0m[90m hides all slides except the first one[0m
[2K[0G    [32m  Γ£ô[0m[90m makes slide with is-active class active initially[0m
[0m    loadBullets()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the bullets[0m
[0m    changeSlide()[0m
[2K[0G    [32m  Γ£ô[0m[90m changes slides to the next one for ltr is true[0m
[2K[0G    [32m  Γ£ô[0m[90m changes slides to the last one for ltr is false[0m
[2K[0G    [32m  Γ£ô[0m[90m changes slides to the chosen one[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m fires beforeslidechange.zf.orbit event[0m
[2K[0G    [32m  Γ£ô[0m[90m fires slidechange.zf.orbit event[0m
[0m    updateBullets()[0m
[2K[0G    [32m  Γ£ô[0m[90m updates the bullets[0m
[0m    events()[0m
[2K[0G    [32m  Γ£ô[0m[90m changes slides on bullet click[0m
[2K[0G    [32m  Γ£ô[0m[90m changes slides to the previous one[0m
[2K[0G    [32m  Γ£ô[0m[90m changes slides to the next one[0m
[0m    geoSync()[0m
[2K[0G    [32m  Γ£ô[0m[90m changes slides automatically based on timerDelay option[0m[31m (213ms)[0m
[0m    keyboard events[0m
[2K[0G    [32m  Γ£ô[0m[90m moves switches to next slide using ARROW_RIGHT[0m
[2K[0G    [32m  Γ£ô[0m[90m moves switches to previous slide using ARROW_LEFT[0m

[0m  Reveal[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m sets ARIA attributes for modal[0m
[2K[0G    [32m  Γ£ô[0m[90m detects anchor if one exists[0m
[2K[0G    [32m  Γ£ô[0m[90m sets ARIA attributes for anchor if one exists[0m
[0m    makeOverlay()[0m
[2K[0G    [32m  Γ£ô[0m[90m creates an overlay if overlay option is true[0m
[0m    open()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the modal[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the overlay[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m adds class to body[0m
[2K[0G    [32m  Γ£ô[0m[90m adds optional overlay classes overlay element[0m
[2K[0G    [32m  Γ£ô[0m[90m fires open.zf.reveal event[0m
[2K[0G    [32m  Γ£ô[0m[90m traps focus if trapFocus option is true[0m
[0m    close()[0m
[2K[0G    [32m  Γ£ô[0m[90m closes the modal[0m
[2K[0G    [32m  Γ£ô[0m[90m closes the overlay[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m removes class from body[0m
[2K[0G    [32m  Γ£ô[0m[90m does not remove class from body if another reveal is open[0m
[2K[0G    [32m  Γ£ô[0m[90m fires closed.zf.reveal event[0m
[2K[0G    [32m  Γ£ô[0m[90m releases focus if trapFocus option is true[0m
[2K[0G    [32m  Γ£ô[0m[90m sets focus to anchor[0m[33m (40ms)[0m
[2K[0G    [32m  Γ£ô[0m[90m sets focus to anchor that opened it[0m[33m (62ms)[0m
[0m    toggle()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens a closed modal[0m
[2K[0G    [32m  Γ£ô[0m[90m closes an open modal[0m
[0m    events()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the modal on anchor click[0m
[2K[0G    [32m  Γ£ô[0m[90m closes a modal on overlay click if closeOnClick option is true[0m
[2K[0G    [32m  Γ£ô[0m[90m not closes a modal on overlay click if closeOnClick option is true[0m
[2K[0G    [32m  Γ£ô[0m[90m closes Reveal with click on close button[0m

[0m  Slider[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores handle, inout and fill elements[0m
[2K[0G    [32m  Γ£ô[0m[90m stores two handle and input elements for two sided slider[0m
[2K[0G    [32m  Γ£ô[0m[90m is disabled when disable option is true[0m
[0m    setHandlePos()[0m
[2K[0G    [32m  Γ£ô[0m[90m positions the handle[0m
[2K[0G    [32m  Γ£ô[0m[90m does nothing if disabled option is true[0m
[2K[0G    [32m  Γ£ô[0m[90m fires changed.zf.slider event[0m[31m (136ms)[0m
[2K[0G    [32m  Γ£ô[0m[90m fires moved.zf.slider event[0m[31m (86ms)[0m
[0m    setValues()[0m
[2K[0G    [32m  Γ£ô[0m[90m updates ARIA attributes[0m
[2K[0G    [32m  Γ£ô[0m[90m updates input value[0m
[0m    setInitAttr()[0m
[2K[0G    [32m  Γ£ô[0m[90m adds ARIA attributes to handle[0m
[2K[0G    [32m  Γ£ô[0m[90m adds attributes to input[0m
[0m    keyboard events[0m
[2K[0G    [32m  Γ£ô[0m[90m sets value to minimum using HOME[0m
[2K[0G    [32m  Γ£ô[0m[90m increases value by step size on ARROW_RIGHT[0m
[2K[0G    [32m  Γ£ô[0m[90m increases value by step size on ARROW_UP[0m
[2K[0G    [32m  Γ£ô[0m[90m decreases value by step size on ARROW_LEFT[0m
[2K[0G    [32m  Γ£ô[0m[90m decreases value by step size on ARROW_DOWN[0m
[2K[0G    [32m  Γ£ô[0m[90m decreases value by step size times 10 on SHIFT_ARROW_RIGHT[0m
[2K[0G    [32m  Γ£ô[0m[90m decreases value by step size times 10 on SHIFT_ARROW_LEFT[0m

[0m  Tabs[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m sets ARIA attributes[0m
[0m    selectTab()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the selected tab via jQuery element[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the selected tab via id string[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the selected tab with data-tabs-target attribute[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the selected tab with data-tabs-target attribute and external href[0m
[0m    _handleTabChange()[0m
[2K[0G    [32m  Γ£ô[0m[90m opens the selected tab[0m
[2K[0G    [32m  Γ£ô[0m[90m sets ARIA attributes for open tab[0m
[2K[0G    [32m  Γ£ô[0m[90m hides the open tab[0m
[2K[0G    [32m  Γ£ô[0m[90m sets ARIA attributes for closed tab[0m
[2K[0G    [32m  Γ£ô[0m[90m fires change.zf.tabs event with target as data[0m
[0m    keyboard events[0m
[2K[0G    [32m  Γ£ô[0m[90m switches to next tab on ARROW_RIGHT[0m
[2K[0G    [32m  Γ£ô[0m[90m switches to next tab on ARROW_DOWN[0m
[2K[0G    [32m  Γ£ô[0m[90m switches to previous tab on ARROW_LEFT[0m
[2K[0G    [32m  Γ£ô[0m[90m switches to previous tab on ARROW_UP[0m

[0m  Toggler[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the class defined on the data-toggler attribute[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the class defined on the data-toggler attribute (with leading dot)[0m
[2K[0G    [32m  Γ£ô[0m[90m stores defined animation classes[0m
[2K[0G    [32m  Γ£ô[0m[90m adds Aria attributes to click triggers[0m
[2K[0G    [32m  Γ£ô[0m[90m sets aria-expanded to true if the element is visible[0m
[2K[0G    [32m  Γ£ô[0m[90m sets aria-expanded to false if the element is invisible[0m
[0m    toggle()[0m
    [36m  - calls Toggler._toggleClass() if the element toggles with a class[0m
    [36m  - calls Toggler._toggleAnimate() if the element toggles with animation[0m
[0m    toggleClass()[0m
[2K[0G    [32m  Γ£ô[0m[90m toggles a class on the element[0m
[2K[0G    [32m  Γ£ô[0m[90m updates aria-expanded after the class is toggled[0m
[0m    toggleAnimate()[0m
    [36m  - animates an invisible element in[0m
    [36m  - animates an visible element out[0m

[0m  Tooltip[0m
[0m    constructor()[0m
[2K[0G    [32m  Γ£ô[0m[90m stores the element and plugin options[0m
[0m    init()[0m
[2K[0G    [32m  Γ£ô[0m[90m has value of title attribute as content[0m
[2K[0G    [32m  Γ£ô[0m[90m has value of tipText option as content[0m
[2K[0G    [32m  Γ£ô[0m[90m uses value of template option as template[0m
[2K[0G    [32m  Γ£ô[0m[90m uses value of triggerClass option as trigger class[0m
[2K[0G    [32m  Γ£ô[0m[90m sets ARIA attributes[0m
[0m    buildTemplate()[0m
[2K[0G    [32m  Γ£ô[0m[90m uses value of templateClasses option as template class[0m
[0m    show()[0m
[2K[0G    [32m  Γ£ô[0m[90m shows the tooltip[0m
[2K[0G    [32m  Γ£ô[0m[90m fires show.zf.tooltip event[0m
[0m    hide()[0m
[2K[0G    [32m  Γ£ô[0m[90m hides the tooltip[0m
[2K[0G    [32m  Γ£ô[0m[90m fires hide.zf.tooltip event[0m
[0m    toggle()[0m
[2K[0G    [32m  Γ£ô[0m[90m shows a hidden tooltip[0m
[2K[0G    [32m  Γ£ô[0m[90m hides a visible tooltip[0m

[0m  Foundation box[0m
[0m    GetDimensions()[0m
[2K[0G    [32m  Γ£ô[0m[90m should be unable to get dimensions for window[0m
[2K[0G    [32m  Γ£ô[0m[90m should be unable to get dimensions for document[0m
[2K[0G    [32m  Γ£ô[0m[90m height and width of element[0m
[2K[0G    [32m  Γ£ô[0m[90m parent height of element[0m

[0m  Foundation core[0m
[2K[0G  [32m  Γ£ô[0m[90m exists on the window[0m
[2K[0G  [32m  Γ£ô[0m[90m is a jQuery prototype function[0m
[0m    rtl()[0m
[2K[0G    [32m  Γ£ô[0m[90m detects the text direction on the document[0m
[0m    plugin()[0m
[2K[0G    [32m  Γ£ô[0m[90m adds Foundation plugins[0m
[2K[0G    [32m  Γ£ô[0m[90m uses the name of the Plugin class/function if one is not provided[0m
[0m    registerPlugin()[0m
    [36m  - registers a new instance of a plugin[0m
[0m    unregisterPlugin()[0m
    [36m  - un-registers a plugin being destroyed[0m
[0m    GetYoDigits()[0m
[2K[0G    [32m  Γ£ô[0m[90m generates a random ID matching a given length[0m
[2K[0G    [32m  Γ£ô[0m[90m can append a namespace to the number[0m
[0m    RegExpEscape()[0m
[2K[0G    [32m  Γ£ô[0m[90m escape all special characters in a string for RegExp[0m

[0m  Keyboard util[0m
[2K[0G  [32m  Γ£ô[0m[90m exists on the Foundation API[0m
[0m    parseKey()[0m
[2K[0G    [32m  Γ£ô[0m[90m returns the character pressed for a normal key[0m
[2K[0G    [32m  Γ£ô[0m[90m returns the character pressed for special keys[0m
[2K[0G    [32m  Γ£ô[0m[90m recognizes if CTRL was pressed[0m
[2K[0G    [32m  Γ£ô[0m[90m recognizes if ALT was pressed[0m
[2K[0G    [32m  Γ£ô[0m[90m recognizes if SHIFT was pressed[0m
[2K[0G    [32m  Γ£ô[0m[90m recognizes if multiple modifiers were pressed[0m
[0m    handleKey()[0m
[2K[0G    [32m  Γ£ô[0m[90m executes callback for given key event[0m
[2K[0G    [32m  Γ£ô[0m[90m executes handled callback for given key event[0m
[2K[0G    [32m  Γ£ô[0m[90m executes unhandled callback for given key event[0m
[0m    findFocusable()[0m
[2K[0G    [32m  Γ£ô[0m[90m finds focusable elements inside a container[0m
[2K[0G    [32m  Γ£ô[0m[90m does not find hidden focusable elements[0m
[2K[0G    [32m  Γ£ô[0m[90m does not find disabled focusable elements[0m
[2K[0G    [32m  Γ£ô[0m[90m does not find focusable elements with negative tabindex[0m
[0m    trapFocus()[0m
[2K[0G    [32m  Γ£ô[0m[90m moves the focus to the first focusable element[0m
[2K[0G    [32m  Γ£ô[0m[90m moves the focus to the last focusable element[0m
[0m    releaseFocus()[0m
[2K[0G    [32m  Γ£ô[0m[90m stops trapping the focus at the end[0m
[2K[0G    [32m  Γ£ô[0m[90m stops trapping the focus at the top[0m


[92m [0m[32m 241 passing[0m[90m (6s)[0m
[36m [0m[36m 8 pending[0m
[31m  2 failing[0m

[0m  1) Abide addA11yAttributes() adds [aria-describedby] attribute to field and [for] attribute to form error:
[0m[31m     AssertionError: expected { Object (length, prevObject, ...) } to have a 'for' attribute[0m[90m
      at file:///C:/git/foundation-sites/test/javascript/lib/chai.js:874
      at file:///C:/git/foundation-sites/test/javascript/lib/chai-jquery.js:66
      at file:///C:/git/foundation-sites/test/javascript/lib/chai.js:4027
      at file:///C:/git/foundation-sites/test/javascript/js-tests.js:112
      at callFn (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4202)
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4195
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4661
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4790
      at next (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4581)
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4591
      at next (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4523)
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4559
      at timeslice (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:12326)
[0m
[0m  2) Abide addA11yAttributes() adds attributes and ids when no id is set:
[0m[31m     AssertionError: expected { Object (length, prevObject, ...) } to have a 'id' attribute[0m[90m
      at file:///C:/git/foundation-sites/test/javascript/lib/chai.js:874
      at file:///C:/git/foundation-sites/test/javascript/lib/chai-jquery.js:66
      at file:///C:/git/foundation-sites/test/javascript/lib/chai.js:4027
      at file:///C:/git/foundation-sites/test/javascript/js-tests.js:125
      at callFn (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4202)
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4195
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4661
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4790
      at next (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4581)
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4591
      at next (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4523)
      at file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:4559
      at timeslice (file:///C:/git/foundation-sites/test/javascript/lib/mocha.js:12326)
[0m



npm
 
ERR!
 
code
 ELIFECYCLE

npm
 
ERR!
 
errno
 2

npm
 
ERR!
 foundation-sites@6.4.3 test:javascript:phantomjs: `npm run 
test:javascript:transpile && mocha-phantomjs --ignore-resource-errors 
test/javascript/index.html`

npm
 
ERR!
 Exit status 2

npm
 
ERR!
 

npm
 
ERR!
 Failed at the foundation-sites@6.4.3 test:javascript:phantomjs script.

npm
 
ERR!
 This is probably not a problem with npm. There is likely additional logging 
output above.



npm
 
ERR!
 A complete log of this run can be found in:

npm
 
ERR!
     C:\Users\chris.crutchfield\AppData\Roaming\npm-cache\_logs\2018-02-13T01_0
8_38_331Z-debug.log

npm
 
ERR!
 Test failed.  See above for more details.
~~~

#### Browser(s) and Device(s) tested on:

* Node.js v8.9.4
* Windows 7 64 bit

#### Foundation Version(s) you are using:

* develop branch.

--- a/js/foundation.abide.js
+++ b/js/foundation.abide.js
@@ -213,8 +213,8 @@ class Abide extends Plugin {
    */
   addA11yAttributes($el) {
     let $errors = this.findFormError($el);
-    let $labels = $errors.filter('label').end();
-    let $error = $errors.first().end();
+    let $labels = $errors.filter('label');
+    let $error = $errors.first();
     if (!$errors.length) return;
 
     // Set [aria-describedby] on the input toward the first form error if it is not set
@@ -229,7 +229,7 @@ class Abide extends Plugin {
       $el.attr('aria-describedby', errorId);
     }
 
-    if ($labels.filter('[for]').end().length < $labels.length) {
+    if ($labels.filter('[for]').length < $labels.length) {
       // Get the input ID or create one
       let elemId = $el.attr('id');
       if (typeof elemId === 'undefined') {
@@ -242,7 +242,7 @@ class Abide extends Plugin {
         const $label = $(label);
         if (typeof $label.attr('for') === 'undefined')
           $label.attr('for', elemId);
-      }).end();
+      });
     }
 
     // For each error targeting $el, set [role=alert] if it is not set.

